src/main/webapp/script.js

```diff
@@ -43,5 +43,92 @@ function updateProjectDatabase() {
 function getJobsFromProject(projectID) {
   fetch('/jobs?projectID=' + projectID)
   .then(response => response.json())
-  .then(jobs => console.log(jobs))
+  .then(jobs => {
+      console.log(jobs);
+      const table = document.getElementById('jobs');
+      for (let i = 0; i < jobs.length; i++) {
+          addJobToTable(jobs[i], table);
+      }
+      table.hidden = false;
```

Comment:
I might have missed something, but is the table hidden at any point?

Reply:
Yes, the table is hidden before pressing ShowInfo button. Hope this answer your question :D

Reply:
Okay thanks - yes it does :)